oedatamodel/latest/v100/datapackage/oedatamodel-readable_datapackage.json

```diff
@@ -0,0 +1,160 @@
+            "path": "oedatamodel-readable_timeseries.csv",
+            "format": "csv",
+            "encoding" : "UTF-8",
+            "schema": {
+                "fields": [
+                    {"name": "scalar_id", "description": "Unique identifier", "type": "bigint", "unit": null},
+                    {"name": "scenario_id", "description": "Scenario name", "type": "text", "unit": null},
+                    {"name": "region", "description": "Country or region", "type": "json", "unit": null},
+                    {"name": "input_energy_vector", "description": "", "type": "integer", "unit": null},
+                    {"name": "output_energy_vector", "description": "", "type": "text", "unit": null},
+                    {"name": "parameter_name", "description": "", "type": "text", "unit": null},
+                    {"name": "technology", "description": "", "type": "text", "unit": null},
+                    {"name": "technology_type", "description": "", "type": "text", "unit": null},
+                    {"name": "unit", "description": "Parameter unit", "type": "text", "unit": null},
+                    {"name": "timeindex start", "description": "Start timestemp", "type": "timestamp", "unit": null},
+                    {"name": "timeindex stop", "description": "Stop timestemp", "type": "timestamp", "unit": null},
+                    {"name": "timeindex resolution", "description": "Timesteps", "type": "intervall", "unit": null},
+                    {"name": "series", "description": "Timesteps", "type": "array[decimal]", "unit": null},
+                    {"name": "tags", "description": "Free classification with key-value pairs", "type": "hstore", "unit": null},
+                    {"name": "method", "description": "Method type (sum, mean, median)", "type": "json", "unit": null},
+                    {"name": "source", "description": "Source", "type": "text", "unit": null},
+                    {"name": "comment", "description": "Comment", "type": "text", "unit": null} ],
+                "primaryKey": ["scalar_id"],
+                "foreignKeys": [{
+                        "fields": ["scenario_id"],
+                        "reference": {
+                            "resource": "oed_scenario",
+                            "fields": ["scenario_id"] } } ] },
+            "dialect":
+                {"delimiter": ";",
+                "decimalSeparator": "."} } ],
+
+"review": {
+    "path": "",
+    "badge": ""},
+"metaMetadata":
+    {"metadataVersion": "OEP-1.4.0",
+    "metadataLicense":
+        {"name": "CC0-1.0",
+        "title": "Creative Commons Zero v1.0 Universal",
+        "path": "https://creativecommons.org/publicdomain/zero/1.0/"} },
+"_comment":
+    {"metadata": "Metadata documentation and explanation (https://github.com/OpenEnergyPlatform/organisation/wiki/metadata)",
+    "dates": "Dates and time must follow the ISO8601 including time zone (YYYY-MM-DD or YYYY-MM-DDThh:mm:ss±hh)",
```

Comment:
I'd suggest to also (explicitly) allow `YYYY` as valid timestamps.

Reply:
thanks for the remark. On the oep this concerns then the [oemetadata](https://github.com/OpenEnergyPlatform/oemetadata), because there the metadata string is developed. I can mention that there.

Reply:
If you just mention it in the oemetadata but leave this comment as is, this might lead to conflicting instructions.